componentLibraries/defaultLibrary/Hydraulic/Volumes&Lines/HydraulicAckumulator.hpp

```diff
@@ -157,21 +158,21 @@ here!)","",0.,&mpxmp);
      {
         //Read port variable pointers from nodes
         //Port P1
-        mpND_p1=getSafeNodeDataPtr(mpP1, NodeHydraulic::Pressure);
-        mpND_q1=getSafeNodeDataPtr(mpP1, NodeHydraulic::Flow);
-        mpND_T1=getSafeNodeDataPtr(mpP1, NodeHydraulic::Temperature);
-        mpND_dE1=getSafeNodeDataPtr(mpP1, NodeHydraulic::HeatFlow);
-        mpND_c1=getSafeNodeDataPtr(mpP1, NodeHydraulic::WaveVariable);
-        mpND_Zc1=getSafeNodeDataPtr(mpP1, NodeHydraulic::CharImpedance);
+        mpP_p1=getSafeNodeDataPtr(mpP1, NodeHydraulic::Pressure);
```

Comment:
Not super important, but it would be even better if they were called mpP1_varName,
Then it matches the latest naming suggestion for this type of variables.

Essentially mpPortName_VariableName